azure/lerna-package-lock.json

```diff
@@ -380,16 +380,6 @@
 				"is-negated-glob": "^1.0.0"
 			}
 		},
-		"@es-joy/jsdoccomment": {
```

Comment:
The removals here suggest that this change results in less plugins being used.  The PR description implies this is just changing how we depend on them. Is this an intentional change to which plugins are used in this sub-monorepo or something else?

Reply:
Some of these plugins were completely unused, I suspect because the base config changed, but the consumers were not correctly updated - one of the downsides of non-local dependencies. I'll make note of that in the description.